### PR TITLE
solution for isl version mismatch

### PR DESCRIPTION
**Pull request by darthburrito**
Update the faq for isl version mismatch 

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/faq_build.md [hunk 1]
```diff
@@ -134,7 +134,23 @@ brew link --force avr-gcc@8
...
+  Referenced from: /usr/local/Cellar/avr-gcc@8/8.3.0/libexec/gcc/avr/8.3.0/cc1
+  Reason: image not found
+avr-gcc: internal compiler error: Abort trap: 6 signal terminated program cc1
+```
+
+The solution is to install the lastest version and update the link.
+
+```
+brew install isl 
+install_name_tool -change '/usr/local/opt/isl/lib/libisl.21.dylib' /usr/local/opt/isl/lib/libisl.dylib /usr/local/Cellar/avr-gcc@8/8.3.0/libexec/gcc/avr/8.3.0/cc1
```
Comment (drashna) [inline docs/faq_build.md]: This doesn't link lto1, though.

```suggestion
install_name_tool -change '/usr/local/opt/isl/lib/libisl.21.dylib' /usr/local/opt/isl/lib/libisl.dylib /usr/local/Cellar/avr-gcc@8/8.3.0/libexec/gcc/avr/8.3.0/cc1
install_name_tool -change '/usr/local/opt/isl/lib/libisl.21.dylib' /usr/local/opt/isl/lib/libisl.dylib /usr/local/Cellar/avr-gcc@8/8.3.0/libexec/gcc/avr/8.3.0/lto1
```

THis should.

**Comment by drashna** [comment]
@fauxpark, is there a good way to automate this too, until there is a fix?

**Comment by fauxpark** [comment]
I would say just tack it on to the end of the install script, after avr-gcc is installed.

**Comment by drashna** [comment]
This has been fixed in the cask for avr-gcc.   

Did we still want to close the issue?

**Comment by fauxpark** [comment]
Seems the root cause was CI on the osx-cross end not rebuilding the package when ISL broke it, per https://github.com/osx-cross/homebrew-avr/issues/136#issuecomment-570887327. Now that seems to have been fixed, so yeah, I think this can be closed.

**Pull request closed without merging**